### DISCUSSION TITLE
Clearing out emails from DB and filesystem.

### DISCUSSION THREAD

**Discussion by splice33** [YetiForceCompany/YetiForceCRM#14208] [How to]
Hi All,

I have a small system that's mostly just used to track communications with customers.

We recently changed email servers, so I updated the email configuration to point to the new server name. I had to go through the process of setting auto-logon and the email scanner again. But... this resulted in redownloading all users mail again. Deleting all the emails from within YetiForce seams to clear them from the database, but not from the filesystem ../storage/ directory. I've had to download all emails again for a third time and the ../storage/ directory is huge. Is there a way to remove files from the storage directory that are no longer needed? 

Thanks!
Paul.

**Comment by rskrzypczak** [comment]
Delete old documents from emails
![image](https://user-images.githubusercontent.com/10221999/107181678-a57b2f00-69db-11eb-9bdb-bdd6b039ab8e.png)

**Reply by splice33** [reply]
Perfect thank you!
